docs/REST-API/20-Alert-Grouping-Settings-API.md

```diff
@@ -0,0 +1,35 @@
+---
+tags: [rest-api]
+---
+
+# Alert Grouping Settings API - FAQ 
+
+
+### What is the maximum number of services for a Global Alert Grouping setting?
+
+With Global Alert Grouping - Content Based, there will be a **limit of 250 services maximum per alert grouping setting.**
+
+
+### What is the difference between the [Services API] endpoints where I set the alert grouping parameters for a given service vs. the [Alert Grouping Settings API] endpoints?
+
+- Existing `/services` endpoints only allow you to define Single Service Alert Grouping.
+- The new `/alert_grouping_settings` endpoints allow you to define Single or Global Content Based Alert Grouping.
+- Single Service Time and Single Intelligent Grouping types can still be set using the `/services` endpoints.
+
+
+### When can I use only the new [Alert Grouping Settings API] endpoints?
```

Comment:
```suggestion
### Can I use only the new [Alert Grouping Settings API] endpoints?
```

Reply:
I'm not a technical writer, but this answer doesn't tell me _when_ I can use only the new endpoints, just that I will be able to in "the future".